include/openrave/posturedescriber.h

```diff
@@ -0,0 +1,116 @@
+// -*- coding: utf-8 -*-
+// Copyright (C) 2006-2020 Guangning Tan, Kei Usui, Rosen Diankov <rosen.diankov@gmail.com>
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#ifndef OPENRAVE_POSTUREDESCRIBER_H
+#define OPENRAVE_POSTUREDESCRIBER_H
+
+#include <openrave/openrave.h>
+
+namespace OpenRAVE {
+
+using LinkPair = RobotBase::LinkPair; ///< a baselink-eelink pair
```

Comment:
@tgn3000 posture should be a property of KinBody too, so why not `KinBody::LinkPair`?